### ISSUE TITLE
Vim mode: support paragraph operation using ap/ip

### ISSUE THREAD

**Issue by noritada** [zed-industries/zed#7359]
### Check for existing issues

- [X] Completed

### Describe the feature

In Vim mode, although users can use `daw` (delete a word), `diw` (delete inner word), `das` (delete a sentence) and `dis` (delete inner sentence), `dap` (delete a paragraph) and `dip` (delete inner paragraph) cannot be used.
It would be nice if it is implemented.

See :help ap in Vim:

```
ap			"a paragraph", select [count] paragraphs (see
			paragraph).
			Exception: a blank line (only containing white space)
			is also a paragraph boundary.
			When used in Visual mode it is made linewise.

							v_ip ip
ip			"inner paragraph", select [count] paragraphs (see
			paragraph).
			Exception: a blank line (only containing white space)
			is also a paragraph boundary.
			When used in Visual mode it is made linewise.
```

**Comment by noritada** [comment]
I read the source code, found the problem area and have started implementation, hoping to be able to submit a PR.

**Comment by ConradIrwin** [comment]
@noritada thank you! If you'd like to pair with me to help get it over the line, feel free to book time: https://calendly.com/conradirwin/pairing

**Comment by noritada** [comment]
@ConradIrwin I have submitted a PR #7687. I hope you will take a look at it when you have time.

**Comment by JosephTLyons** [comment]
This is available in Zed [v0.126.0-pre](https://zed.dev/releases/preview/0.126.0).
